src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglx_utils.py

```diff
@@ -7,30 +7,6 @@
 from pydantic import FilePath
 
 
-def add_recording_extractor_properties(recording_extractor) -> None:
```

Comment:
Moving away from the utils pattern here. I don't really want to have such a critical behavior (renaming the channel names) hidden away in a util.